tools/cooja/java/org/contikios/cooja/radiomediums/AbstractRadioMedium.java

```diff
@@ -236,8 +236,9 @@ public void update(Observable obs, Object obj) {
 				case RECEPTION_STARTED:
 				case RECEPTION_INTERFERED:
 				case RECEPTION_FINISHED:
+					break;
+
 				case UNKNOWN:
-					return;
```

Comment:
I believe that mspsim fires an `UNKOWN` event when channel is changed in cc2420
https://github.com/contiki-os/contiki/blob/master/tools/cooja/apps/mspsim/src/org/contikios/cooja/mspmote/interfaces/Msp802154Radio.java#L159